x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/action/DeleteDataFrameAnalyticsAction.java

```diff
@@ -30,13 +30,21 @@ private DeleteDataFrameAnalyticsAction() {
         super(NAME, AcknowledgedResponse::new);
     }
 
-    public static class Request extends AcknowledgedRequest<Request> implements ToXContentFragment {
+    public static class Request extends AcknowledgedRequest<Request> {
+
+        public static final ParseField FORCE = new ParseField("force");
 
         private String id;
+        private boolean force;
 
         public Request(StreamInput in) throws IOException {
             super(in);
             id = in.readString();
+            if (in.getVersion().onOrAfter(Version.CURRENT)) {
```

Comment:
#nit, 
```suggestion
            if (in.getVersion().onOrAfter(Version.V_8_0_0)) {
```

Reply:
I will have to turn off BWC tests for this. I have the habit of leaving those as `CURRENT` and change them when I unmute the BWC tests after backporting.

Reply:
@dimitris-athanasiou I can understand that, I have just noticed that using `CURRENT` allows you to backport without actually changing the version (since it still compiles). `V_8_0_0` forces me to do things in the correct order.

I just find this helpful, I am not gonna prevent a 👍 over it :D